views/layouts/applicationLayout.sejs

```diff
@@ -195,6 +195,13 @@
     <div id="alert-socket-disconnect" class="alert alert-warning" role="alert">
       <strong>Warning: lost connection. Please try refreshing this page or check your internet connection.</strong>
     </div>
+
+    <% if(process.env.CSMM_DONOR_ONLY === 'true'){ %>
+      <div class="alert alert-warning" role="alert">
+        <strong>(!) Free hosting is being discontinued - join <a href="https://catalysm.net/discord">the Discord</a> for more information</strong>
```

Comment:
Again, can you make sure the only way to get info isn't just discord? I'm sure some don't want to sign up.

Reply:
Ohh I didnt realize that's what you meant. Should we add a full explanation to the docs then? Probably related: https://github.com/CatalysmsServerManager/docs/pull/246